### DISCUSSION TITLE
How do I use leap with delete?

### DISCUSSION THREAD

**Discussion by rkk1995** [ggandor/leap.nvim#157] [Q&A] [answered]
Say I want to delete to a certain letter. How can I do this with leap?

With lightspeed I could do `d + z + <character I'm interested in>`

**Comment by rkk1995** [comment, marked as answer]
Nevermind, I had another plugin overwritting `s` whenever I entered delete mode.

**Comment by gyaaniguy** [comment]
Edit: figured it out. it was vim-surround. For googlers use `:map ds` to find the existing hotkey
I think I might have the same issue.

Can someone confirm , if the default behaviour is to delete with `d + s + <chars to delete till>`  
@rkk1995 Can you share which plugin was overwriting the 's' key in your case.
